src/logging.h

```diff
@@ -54,6 +54,7 @@ namespace BCLog {
         COINDB      = (1 << 18),
         QT          = (1 << 19),
         LEVELDB     = (1 << 20),
+        RPCPARAMS   = (1 << 21),
```

Comment:
I'm undecided if a new category is needed, or just enable this additional logging with `rpc`. We don't have many of the 32 category bits remaining (although we could easily change to use a `uint64_t`).